aws-lambda-java-events/src/main/java/com/amazonaws/services/lambda/runtime/graalvm/LambdaEventsFeature.java

```diff
@@ -0,0 +1,95 @@
+package com.amazonaws.services.lambda.runtime.graalvm;
+
+import com.amazonaws.services.lambda.runtime.events.S3Event;
+import com.oracle.svm.core.annotate.AutomaticFeature;
+import org.graalvm.nativeimage.hosted.Feature;
+import org.graalvm.nativeimage.hosted.RuntimeReflection;
+
+import java.io.File;
+import java.io.IOException;
+import java.lang.reflect.Constructor;
+import java.lang.reflect.Field;
+import java.lang.reflect.Method;
+import java.net.URL;
+import java.util.ArrayList;
+import java.util.Enumeration;
+import java.util.List;
+
+@AutomaticFeature
```

Comment:
So `native-image` uses this annotation in the analysis phase to build a list of the event classes so that the customer doesn't have to add them to their reflect-config.json?

Reply:
Yes :)